static/i18n/en/app.json

```diff
@@ -1339,6 +1351,44 @@
       }
     },
     "apps": {
+      "content": {
+        "paraswap": {
+          "shortDescription": "Swap your crypto with ParaSwap that aggregates and provides the best quotes decentralised exchanges.",
```

Comment:
Small grammatical thing: I think this should be 'Swap your crypto with ParaSwap that aggregates and provides the best quotes *on* decentralised exchanges.' I assume this was copied from somewhere so I can ask someone in product to confirm @juan-cortes

Reply:
Yeah, I confirm that's the version we currently have in production.

Reply:
Ok I will raise in the ticket with a product owner

Reply:
I included this fix to the PR.